Jenkinsfile.tab

```diff
@@ -0,0 +1,72 @@
+pipeline {
+  agent{
+    label 'BASIC_SLAVE&&!beefy_BASIC_SLAVE'
+  }
+
+  options {
+    timeout(time: 2, unit: 'HOURS')
+    timestamps()
+    ansiColor('xterm')
+  }
+
+  stages {
+    stage('Checkout') {
+        steps{
+            checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'a8c27fc2-52cd-499c-b648-0355ed5ca72a', url: 'https://github.com/webex/react-ciscospark']]]
```

Comment:
Can we split this into multiple lines so it's easier to read?

Reply:
Will do.